src/Config_Command.php

```diff
@@ -619,6 +619,54 @@ public function has( $args, $assoc_args ) {
 		}
 	}
 
+	/**
+	 * Refreshes the salts defined in the wp-config.php file
+	 *
+	 * ## OPTIONS
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     # Get new salts for your wp-config.php file
+	 *     $ wp config shuffle-salts
+	 *
+	 * @subcommand shuffle-salts
+	 * @when before_wp_load
+	 */
+	public function shuffle_salts( $args, $assoc_args ) {
+
+		try {
+			for ( $i = 0; $i < 8; $i++ ) {
```

Comment:
Please directly couple this to the number of known constants:
```
foreach ( $constant_list as $key ) {
   $secret_keys[ $key ] = self::unique_key();
}
```
You'll have to pull the declaration of `$constant_list` higher up for this.